program/src/processor/deactivate_stake.rs

```diff
@@ -12,13 +12,16 @@ use crate::{
     state::{find_stake_pda, Config, Stake},
 };
 
+const MAX_BASIS_POINTS: u128 = 10_000;
```

Comment:
side note: unless I'm missing something, there's no check during initialize config or update config that the `basis_points <= 10_000`. we should add that in

Reply:
Oh, good catch, definitely need to add that.

Reply:
Done.